From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Wed, 21 Oct 2020 03:29:18 +0200
Subject: audioDeviceSelection: Center-align icons

When the icon area gets allocated additional space, we want the
icons centered rather than left-aligned. This may happen in locales
with long-ish translations for the title or buttons when only a
subset of possible icons is shown.

Bug: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3307
Origin: upstream, 3.38.2
---
 js/ui/audioDeviceSelection.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/js/ui/audioDeviceSelection.js b/js/ui/audioDeviceSelection.js
index 1e9cb11..07b8171 100644
--- a/js/ui/audioDeviceSelection.js
+++ b/js/ui/audioDeviceSelection.js
@@ -43,6 +43,7 @@ var AudioDeviceSelectionDialog = GObject.registerClass({
 
         this._selectionBox = new St.BoxLayout({
             style_class: 'audio-selection-box',
+            x_align: Clutter.ActorAlign.CENTER,
             x_expand: true,
         });
         content.add_child(this._selectionBox);
